src/controllers/investmentController.ts

```
@@ -105,18 +105,21 @@ export async function postInvestmentWithdrawRequest(
   }
 }
 
+const WITHDRAWAL_STATUSES = ["requested", "available", "completed", "cancelled"] as const;
```

ESLint

[error] Replace `"requested",·"available",·"completed",·"cancelled"` with `⏎··"requested",⏎··"available",⏎··"completed",⏎··"cancelled",⏎`

(prettier/prettier)